docs/source/spec/core.rst

```diff
@@ -5550,6 +5550,27 @@ Value type
         }
 
 
+.. _unstable-trait:
+
+``_unstable`` trait
+---------------
+
+Summary
+    Indicates a shape is unstable and may change in the future.
```

Comment:
"may" -> "MAY".

I think this should give a bit more guidance after the first sentence:

"This trait can be applied to trait definitions to indicate that a trait is unstable or experimental. If possible, code generators SHOULD use this trait to warn when code generated from unstable features are used."